recipes/torchio/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
This LICENSE file is already included in the PyPI tarball, so go ahead and remove this file from the PR.